Formula/jdupes.rb

```diff
@@ -1,21 +1,19 @@
 class Jdupes < Formula
   desc "Duplicate file finder and an enhanced fork of 'fdupes'"
   homepage "https://github.com/jbruchon/jdupes"
-  url "https://github.com/jbruchon/jdupes/archive/v1.13.2.tar.gz"
-  sha256 "41376ace274320a77e35ffed170ff0cbca2cd0cfd0ff1cea53d7464e97d4341f"
-
-  bottle do
-    cellar :any_skip_relocation
```

Comment:
Please don't delete the bottle block - the bottles will be updated by our CI during the merge process.